lua/src/macro_cache.cc

```diff
@@ -108,6 +109,36 @@ QString const& macro_cache::get_host_name(unsigned int host_id) const {
   return (found->host_name);
 }
 
+
+/**
+ *  Get a list of the host groups containing a host.
+ *
+ *  @param[in] host_id  The id of the host.
+ *
+ *  @return             A QStringList.
+ */
+QMultiHash<unsigned int, neb::host_group_member> const&
+                                   macro_cache::get_host_group_members() const {
+  return _host_group_members;
+}
+
+/**
+ *  Get the name of a host group.
+ *
+ *  @param[in] id  The id of the host group.
+ *
+ *  @return             The name of the host group.
+ */
+QString const& macro_cache::get_host_group_name(unsigned int id) const {
+  QHash<unsigned int, neb::host_group>::const_iterator
+    found(_host_groups.find(id));
+
+  if (found == _host_groups.end())
+    throw (exceptions::msg()
```

Comment:
How are exceptions handled by the Lua interpreter ?

Reply:
They are not.
That's why there is a try /  catch in l_broker_cache functions, so that we can return an error.

Reply:
Perfect :-)